contracts/examples/multisig/src/lib.rs

```diff
@@ -350,6 +350,14 @@ pub trait Multisig {
 
 	fn perform_action(&self, action_id: usize) -> SCResult<MultiResultVec<BoxedBytes>> {
 		let action = self.get_action_data(action_id);
+
+		// clean up storage
+		// happens before actual execution, because the async_call kills contract execution,
+		// so cleanup cannot happen afterwards
+		self.set_action_data(action_id, &Action::Nothing);
+		self.set_action_signer_ids(action_id, &[][..]);
```

Comment:
Use #[storage_clear] instead of manually assigning empty slice

Reply:
It's ok here, I think the serialization in this case in a no-op.